ext/snowcrash/src/ParametersParser.h

```diff
@@ -16,6 +16,8 @@
 #include "StringUtility.h"
 #include "BlueprintUtility.h"
 
+#include "../../../src/parser/Uritemplate.h"
```

Comment:
If we merge this, we should move snowcrash into the `src/parser` directory ASAP so we can avoid this kind of ugliness.